src/cptextras/dlcolors.py

```diff
@@ -75,12 +75,18 @@ def prepare_canvas(tailoring=None,predictand='PRCP',type=None):
                 vmin= 0
                 vmax= 1
                 mark='red'
-                barcolor=cmaps['DL_RAIN_POE_COLORMAP']
+                if user_color == None:
+                    barcolor=cmaps['DL_RAIN_POE_COLORMAP']
+                else:
+                    barcolor=cmaps[user_color]
             elif any(x in predictand for x in ['TMAX','TMIN','TMEAN','TMED']):
                 vmin= 0
                 vmax= 1
                 mark='black'
-                barcolor=cmaps['DL_TEMP_POE_COLORMAP']
+                if user_color == None:
+                    barcolor=cmaps['DL_TEMP_POE_COLORMAP']
+                else:
+                    barcolor=cmaps[user_color]
```

Comment:
You've only enabled `user_color` for `tailoring == 'POE'` and values of `predictand` that contain `TMAX`, `TMIN`, `TMEAN`, OR `TMED`. If `user_color` is provided, shouldn't we respect it no matter what the values of `tailoring` and  `predictand` are?

Reply:
The modification also includes the PRCP predictand (When the tailoring is POE)... I can generalize the user_color option for the other tailorings, I understood that customize option only would be available for POE ..... 

Green light for implementing that for all options?